src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -4,14 +4,34 @@
 import graphql.ExecutionResult;
```

Comment:
There are no code hygeine rules for the project so I let IDEA do its full import behaviour of .* imports.  This is what we use here in Atlassian and I am a fan of it.